tests/integration/modules/test_cp.py

```diff
@@ -481,7 +482,8 @@ def test_hash_file(self):
                 [
                     'salt://grail/scene33',
                 ])
-        with salt.utils.fopen(path, 'r') as fn_:
+        mode = 'rb' if salt.utils.is_windows() else 'r'
+        with salt.utils.fopen(path, mode) as fn_:
             data = fn_.read()
             if six.PY3:
                 data = salt.utils.to_bytes(data)
```

Comment:
@s0undt3ch This is still an issue after your change to `salt/__init__.py`. This is the error:
```
 --------  Failed Tests  ---------------------------------------------
   -> integration.modules.test_cp.CPModuleTest.test_hash_file  .......
       Traceback (most recent call last):
         File "c:\salt-dev\salt\tests\integration\modules\test_cp.py", line 489, in test_hash_file
           sha256_hash['hsum'], hashlib.sha256(data).hexdigest())
       AssertionError: 'dfccc700038574c3f61275860dd074a6e2ddb6eee59693b05dcde3bb41b65b55' != '2689de4b07720aafe8f709619f142021ae8dbdd2742a3c86701cd794f97f506c'
       - dfccc700038574c3f61275860dd074a6e2ddb6eee59693b05dcde3bb41b65b55
       + 2689de4b07720aafe8f709619f142021ae8dbdd2742a3c86701cd794f97f506c
   ...................................................................
```
It's reading the file, but not getting the right hash. If I have it read in with 'rb' then it works.